src/io/mdframe_handler_netcdf.cpp

```diff
@@ -0,0 +1,75 @@
+#include "mdframe.hpp"
+
+#include <netcdf>
+
+namespace io {
+
+namespace visitors { // -------------------------------------------------------
+
+/**
+ * mdarray visitor for putting values from the mdarray into a netCDF var.
+ */
+MDARRAY_VISITOR(mdarray_putvar, void)
+{
+    MDARRAY_VISITOR_TEMPLATE_IMPL(T& v, netCDF::NcVar& var) -> void
+    {
+        for (auto it = v.begin(); it != v.end(); it++) {
+            var.putVar(it.mdindex(), *it);
+        }
+    }
+};
```

Comment:
Is this the only use of `mdarray::iterator`? Are any others anticipated?

If it's the only one, I think I'd prefer just a loop over the linear index range, and explicit calls to `deindex`

Reply:
> Note: this chunk of code is now in `include/utilities/mdframe/handler_netcdf.hpp`

Currently, I think it is, but I think there is a use-case for iterator-based access if the `mdarray` is being used outside of the context of an output handler. I haven't gotten a chance to _find_ that use-case, but I'm currently expecting it

On the other hand, it's quite possible that use-case never appears, and if that's the case then I'll change this over as necessary